### PR TITLE
Add ggplot lesson for Oct 27 2021

### PR DESCRIPTION
**Pull request by vickimzhang**

### CODE REVIEW

File: lessons/r/ggplot2/ggplot2-2021.Rmd [hunk 1]
```diff
@@ -0,0 +1,331 @@
+---
+title: Data visualization in R with ggplot2
+author: Vicki Zhang, borrowed from Ahmed Hasan and James Santangelo, who borrowed from Joel Ostblom
+output: pdf_document
+---
+
```
Comment (vickimzhang) [inline lessons/r/ggplot2/ggplot2-2021.Rmd:6]: test comment

Comment (vickimzhang) [reply lessons/r/ggplot2/ggplot2-2021.Rmd:6]: cool

File: lessons/r/ggplot2/ggplot2-2021.Rmd [hunk 2]
```diff
@@ -0,0 +1,331 @@
+---
+title: Data visualization in R with ggplot2
+author: Vicki Zhang, borrowed from Ahmed Hasan and James Santangelo, who borrowed from Joel Ostblom
```
Comment (aays) [inline lessons/r/ggplot2/ggplot2-2021.Rmd]: 'Vicki Zhang, borrowing from x, y, and z' should be fine before this gets out of hand 😄

Comment (vickimzhang) [reply lessons/r/ggplot2/ggplot2-2021.Rmd]: lol fixing it & then will submit another pr!

Comment (aays) [reply lessons/r/ggplot2/ggplot2-2021.Rmd]: No need for another PR, just add another commit to the `r-ggplot` branch on your fork

File: .Rhistory [hunk 3]
```diff
@@ -0,0 +1,14 @@
+knitr::opts_chunk$set(eval=FALSE)
```
Comment (QuLogic) [inline .Rhistory]: Please don't commit this file; preferably rebase and remove it instead of deleting it in a new commit.

**Pull request merged**